themes/default/content/templates/kubernetes/aws/index.md

```diff
@@ -0,0 +1,144 @@
+---
+title: "Kubernetes Cluster on AWS"
+layout: template
+
+# Make sure this is description accurate for this template.
+meta_desc: The AWS Kubernetes template makes it easy to deploy a Kubernetes cluster on AWS with Pulumi and Amazon EKS.
+
+# Appears on the cards on template-overview pages.
+card_desc: Deploy a Kubernetes cluster on AWS with Pulumi and Amazon EKS.
+
+# Used for generating language-specific links to templates on GitHub. (Example: `static-website-aws`)
+template:
+    prefix: kubernetes-aws
+
+# Used for generating links to sibling templates in the right-hand nav. Slug is this template's parent directory.
+cloud:
+  name: Amazon Web Services
+  slug: aws
+
+# Be sure to replace this image. Figma source file:
+# https://www.figma.com/file/lGrSpwbGGmbixEuewMbtkh/Template-Architecture-Diagrams?node-id=15%3A196
+meta_image: meta.png
+
+# The content below is meant help you get started and to serve as a guide to work by. Feel free to adjust it needed for your template.
+---
+
+The AWS Kubernetes template creates an infrastructure as code project in your favorite language and deploys a managed Kubernetes cluster to AWS. The architecture includes a VPC with public and private subnets and deploys an [Amazon EKS cluster]({{< relref "/registry/packages/eks/api-docs/cluster" >}}) that provides a managed Kubernetes control plane. Kubernetes worker nodes are deployed on private subnets for improved security. Load balancers created by workloads deployed on the EKS cluster will be automatically created in the public subnets. The template generates a complete infrastructure as code program to give you a working project out of the box that you can customize easily and extend to suit your needs.
```

Comment:
Highlighting that you are calling this a Kubernetes Cluster on AWS template as the title which is different than here where you are calling it a AWS Kubernetes template.